operations/mimir/rollout-operator.libsonnet

```diff
@@ -65,7 +65,7 @@
     container.mixin.readinessProbe.httpGet.withPort(8001) +
     container.mixin.readinessProbe.withInitialDelaySeconds(5) +
     container.mixin.readinessProbe.withTimeoutSeconds(1) +
-    $.jaeger_mixin,
+    $.tracing_env_mixin,
```

Comment:
> `rollout_operator_container`

Does rollout-operator support OTLP env also?

Reply:
Good point, if it doesn't, I'll fix it now.

Reply:
Updating rollout-operator in https://github.com/grafana/mimir/pull/11787

Reply:
That was merged.